perl: warning: Setting locale failed.
perl: warning: Please check that your locale settings:
	LANGUAGE = (unset),
	LC_ALL = (unset),
	LANG = "C.UTF-8"
    are supported and installed on your system.
perl: warning: Falling back to the standard locale ("C").
Traceback (most recent call last):
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 263, in run_path
    pkg_name=pkg_name, script_name=fname)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "bingo_evolution.py", line 237, in <module>
    main(model)
  File "bingo_evolution.py", line 219, in main
    execute_generational_steps(model)
  File "bingo_evolution.py", line 206, in execute_generational_steps
    checkpoint_base_name='checkpoint_', num_checkpoints=5,stagnation_generations=STAGNATION)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/util/argument_validation.py", line 84, in do_validation
    return func(*pargs, **kwargs)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/evolutionary_optimizer.py", line 143, in evolve_until_convergence
    self._update_checkpoints(checkpoint_base_name, num_checkpoints)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/evolutionary_optimizer.py", line 200, in _update_checkpoints
    self._remove_stale_checkpoint()
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/parallel_archipelago.py", line 276, in _remove_stale_checkpoint
    os.remove(self._previous_checkpoints.pop(0))
FileNotFoundError: [Errno 2] No such file or directory: 'checkpoint__0.pkl'
starting
done
[0.98058293 0.94767262 0.91831705 ... 0.42409892 0.44771494 0.47939977]
[[0.90909091 0.2        0.02177   ]
 [0.90909091 0.2        0.20964   ]
 [0.90909091 0.2        0.43674   ]
 ...
 [0.5        0.8        2.69279   ]
 [0.5        0.8        2.91488   ]
 [0.5        0.8        3.13022   ]]
(5115, 3)
(5115,)
0.6307210305390171
1.1489833279288122
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 0
[unset]: write_line error; fd=-1 buf=:cmd=abort exitcode=1
:
system msg for write_line failure : Bad file descriptor
srun: error: notch272: task 7: Exited with exit code 1
Traceback (most recent call last):
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 263, in run_path
    pkg_name=pkg_name, script_name=fname)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "bingo_evolution.py", line 237, in <module>
    main(model)
  File "bingo_evolution.py", line 219, in main
    execute_generational_steps(model)
  File "bingo_evolution.py", line 206, in execute_generational_steps
    checkpoint_base_name='checkpoint_', num_checkpoints=5,stagnation_generations=STAGNATION)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/util/argument_validation.py", line 84, in do_validation
    return func(*pargs, **kwargs)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/evolutionary_optimizer.py", line 143, in evolve_until_convergence
    self._update_checkpoints(checkpoint_base_name, num_checkpoints)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/evolutionary_optimizer.py", line 200, in _update_checkpoints
    self._remove_stale_checkpoint()
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/parallel_archipelago.py", line 276, in _remove_stale_checkpoint
    os.remove(self._previous_checkpoints.pop(0))
FileNotFoundError: [Errno 2] No such file or directory: 'checkpoint__0.pkl'
starting
done
[0.98058293 0.94767262 0.91831705 ... 0.42409892 0.44771494 0.47939977]
[[0.90909091 0.2        0.02177   ]
 [0.90909091 0.2        0.20964   ]
 [0.90909091 0.2        0.43674   ]
 ...
 [0.5        0.8        2.69279   ]
 [0.5        0.8        2.91488   ]
 [0.5        0.8        3.13022   ]]
(5115, 3)
(5115,)
0.6307210305390171
1.1489833279288122
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 0
[unset]: write_line error; fd=-1 buf=:cmd=abort exitcode=1
:
system msg for write_line failure : Bad file descriptor
srun: error: notch272: task 3: Exited with exit code 1
starting
done
[0.98058293 0.94767262 0.91831705 ... 0.42409892 0.44771494 0.47939977]
[[0.90909091 0.2        0.02177   ]
 [0.90909091 0.2        0.20964   ]
 [0.90909091 0.2        0.43674   ]
 ...
 [0.5        0.8        2.69279   ]
 [0.5        0.8        2.91488   ]
 [0.5        0.8        3.13022   ]]
(5115, 3)
(5115,)
0.6307210305390171
1.1489833279288122
Traceback (most recent call last):
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 263, in run_path
    pkg_name=pkg_name, script_name=fname)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "bingo_evolution.py", line 237, in <module>
    main(model)
  File "bingo_evolution.py", line 219, in main
    execute_generational_steps(model)
  File "bingo_evolution.py", line 206, in execute_generational_steps
    checkpoint_base_name='checkpoint_', num_checkpoints=5,stagnation_generations=STAGNATION)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/util/argument_validation.py", line 84, in do_validation
    return func(*pargs, **kwargs)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/evolutionary_optimizer.py", line 143, in evolve_until_convergence
    self._update_checkpoints(checkpoint_base_name, num_checkpoints)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/evolutionary_optimizer.py", line 200, in _update_checkpoints
    self._remove_stale_checkpoint()
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/parallel_archipelago.py", line 276, in _remove_stale_checkpoint
    os.remove(self._previous_checkpoints.pop(0))
FileNotFoundError: [Errno 2] No such file or directory: 'checkpoint__0.pkl'
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 0
[unset]: write_line error; fd=-1 buf=:cmd=abort exitcode=1
:
system msg for write_line failure : Bad file descriptor
srun: error: notch272: task 5: Exited with exit code 1
/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evaluation/fitness_function.py:19: RuntimeWarning: overflow encountered in square
  return np.sqrt(np.mean(np.square(vector)))
Traceback (most recent call last):
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 263, in run_path
    pkg_name=pkg_name, script_name=fname)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "bingo_evolution.py", line 237, in <module>
    main(model)
  File "bingo_evolution.py", line 219, in main
    execute_generational_steps(model)
  File "bingo_evolution.py", line 206, in execute_generational_steps
    checkpoint_base_name='checkpoint_', num_checkpoints=5,stagnation_generations=STAGNATION)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/util/argument_validation.py", line 84, in do_validation
    return func(*pargs, **kwargs)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/evolutionary_optimizer.py", line 143, in evolve_until_convergence
    self._update_checkpoints(checkpoint_base_name, num_checkpoints)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/evolutionary_optimizer.py", line 200, in _update_checkpoints
    self._remove_stale_checkpoint()
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/parallel_archipelago.py", line 276, in _remove_stale_checkpoint
    os.remove(self._previous_checkpoints.pop(0))
FileNotFoundError: [Errno 2] No such file or directory: 'checkpoint__0.pkl'
starting
done
[0.98058293 0.94767262 0.91831705 ... 0.42409892 0.44771494 0.47939977]
[[0.90909091 0.2        0.02177   ]
 [0.90909091 0.2        0.20964   ]
 [0.90909091 0.2        0.43674   ]
 ...
 [0.5        0.8        2.69279   ]
 [0.5        0.8        2.91488   ]
 [0.5        0.8        3.13022   ]]
(5115, 3)
(5115,)
0.6307210305390171
1.1489833279288122
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 0
[unset]: write_line error; fd=-1 buf=:cmd=abort exitcode=1
:
system msg for write_line failure : Bad file descriptor
srun: error: notch272: task 10: Exited with exit code 1
starting
done
[0.98058293 0.94767262 0.91831705 ... 0.42409892 0.44771494 0.47939977]
[[0.90909091 0.2        0.02177   ]
 [0.90909091 0.2        0.20964   ]
 [0.90909091 0.2        0.43674   ]
 ...
 [0.5        0.8        2.69279   ]
 [0.5        0.8        2.91488   ]
 [0.5        0.8        3.13022   ]]
(5115, 3)
(5115,)
0.6307210305390171
1.1489833279288122
Traceback (most recent call last):
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 263, in run_path
    pkg_name=pkg_name, script_name=fname)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "bingo_evolution.py", line 237, in <module>
    main(model)
  File "bingo_evolution.py", line 219, in main
    execute_generational_steps(model)
  File "bingo_evolution.py", line 206, in execute_generational_steps
    checkpoint_base_name='checkpoint_', num_checkpoints=5,stagnation_generations=STAGNATION)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/util/argument_validation.py", line 84, in do_validation
    return func(*pargs, **kwargs)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/evolutionary_optimizer.py", line 143, in evolve_until_convergence
    self._update_checkpoints(checkpoint_base_name, num_checkpoints)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/evolutionary_optimizer.py", line 200, in _update_checkpoints
    self._remove_stale_checkpoint()
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/parallel_archipelago.py", line 276, in _remove_stale_checkpoint
    os.remove(self._previous_checkpoints.pop(0))
FileNotFoundError: [Errno 2] No such file or directory: 'checkpoint__0.pkl'
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 0
[unset]: write_line error; fd=-1 buf=:cmd=abort exitcode=1
:
system msg for write_line failure : Bad file descriptor
srun: error: notch272: task 9: Exited with exit code 1
starting
done
[0.98058293 0.94767262 0.91831705 ... 0.42409892 0.44771494 0.47939977]
[[0.90909091 0.2        0.02177   ]
 [0.90909091 0.2        0.20964   ]
 [0.90909091 0.2        0.43674   ]
 ...
 [0.5        0.8        2.69279   ]
 [0.5        0.8        2.91488   ]
 [0.5        0.8        3.13022   ]]
(5115, 3)
(5115,)
0.6307210305390171
1.1489833279288122
Traceback (most recent call last):
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 263, in run_path
    pkg_name=pkg_name, script_name=fname)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "bingo_evolution.py", line 237, in <module>
    main(model)
  File "bingo_evolution.py", line 219, in main
    execute_generational_steps(model)
  File "bingo_evolution.py", line 206, in execute_generational_steps
    checkpoint_base_name='checkpoint_', num_checkpoints=5,stagnation_generations=STAGNATION)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/util/argument_validation.py", line 84, in do_validation
    return func(*pargs, **kwargs)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/evolutionary_optimizer.py", line 143, in evolve_until_convergence
    self._update_checkpoints(checkpoint_base_name, num_checkpoints)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/evolutionary_optimizer.py", line 200, in _update_checkpoints
    self._remove_stale_checkpoint()
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/parallel_archipelago.py", line 276, in _remove_stale_checkpoint
    os.remove(self._previous_checkpoints.pop(0))
FileNotFoundError: [Errno 2] No such file or directory: 'checkpoint__0.pkl'
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 0
[unset]: write_line error; fd=-1 buf=:cmd=abort exitcode=1
:
system msg for write_line failure : Bad file descriptor
srun: error: notch272: task 1: Exited with exit code 1
starting
done
[0.98058293 0.94767262 0.91831705 ... 0.42409892 0.44771494 0.47939977]
[[0.90909091 0.2        0.02177   ]
 [0.90909091 0.2        0.20964   ]
 [0.90909091 0.2        0.43674   ]
 ...
 [0.5        0.8        2.69279   ]
 [0.5        0.8        2.91488   ]
 [0.5        0.8        3.13022   ]]
(5115, 3)
(5115,)
0.6307210305390171
1.1489833279288122
Traceback (most recent call last):
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 263, in run_path
    pkg_name=pkg_name, script_name=fname)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "bingo_evolution.py", line 237, in <module>
    main(model)
  File "bingo_evolution.py", line 219, in main
    execute_generational_steps(model)
  File "bingo_evolution.py", line 206, in execute_generational_steps
    checkpoint_base_name='checkpoint_', num_checkpoints=5,stagnation_generations=STAGNATION)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/util/argument_validation.py", line 84, in do_validation
    return func(*pargs, **kwargs)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/evolutionary_optimizer.py", line 143, in evolve_until_convergence
    self._update_checkpoints(checkpoint_base_name, num_checkpoints)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/evolutionary_optimizer.py", line 200, in _update_checkpoints
    self._remove_stale_checkpoint()
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/parallel_archipelago.py", line 276, in _remove_stale_checkpoint
    os.remove(self._previous_checkpoints.pop(0))
FileNotFoundError: [Errno 2] No such file or directory: 'checkpoint__0.pkl'
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 0
[unset]: write_line error; fd=-1 buf=:cmd=abort exitcode=1
:
system msg for write_line failure : Bad file descriptor
srun: error: notch272: task 6: Exited with exit code 1
/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/numpy/core/_methods.py:178: RuntimeWarning: overflow encountered in reduce
  ret = umr_sum(arr, axis, dtype, out, keepdims, where=where)
Traceback (most recent call last):
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 263, in run_path
    pkg_name=pkg_name, script_name=fname)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "bingo_evolution.py", line 237, in <module>
    main(model)
  File "bingo_evolution.py", line 219, in main
    execute_generational_steps(model)
  File "bingo_evolution.py", line 206, in execute_generational_steps
    checkpoint_base_name='checkpoint_', num_checkpoints=5,stagnation_generations=STAGNATION)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/util/argument_validation.py", line 84, in do_validation
    return func(*pargs, **kwargs)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/evolutionary_optimizer.py", line 143, in evolve_until_convergence
    self._update_checkpoints(checkpoint_base_name, num_checkpoints)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/evolutionary_optimizer.py", line 200, in _update_checkpoints
    self._remove_stale_checkpoint()
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/parallel_archipelago.py", line 276, in _remove_stale_checkpoint
    os.remove(self._previous_checkpoints.pop(0))
FileNotFoundError: [Errno 2] No such file or directory: 'checkpoint__0.pkl'
starting
done
[0.98058293 0.94767262 0.91831705 ... 0.42409892 0.44771494 0.47939977]
[[0.90909091 0.2        0.02177   ]
 [0.90909091 0.2        0.20964   ]
 [0.90909091 0.2        0.43674   ]
 ...
 [0.5        0.8        2.69279   ]
 [0.5        0.8        2.91488   ]
 [0.5        0.8        3.13022   ]]
(5115, 3)
(5115,)
0.6307210305390171
1.1489833279288122
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 0
[unset]: write_line error; fd=-1 buf=:cmd=abort exitcode=1
:
system msg for write_line failure : Bad file descriptor
srun: error: notch272: task 0: Exited with exit code 1
starting
done
[0.98058293 0.94767262 0.91831705 ... 0.42409892 0.44771494 0.47939977]
[[0.90909091 0.2        0.02177   ]
 [0.90909091 0.2        0.20964   ]
 [0.90909091 0.2        0.43674   ]
 ...
 [0.5        0.8        2.69279   ]
 [0.5        0.8        2.91488   ]
 [0.5        0.8        3.13022   ]]
(5115, 3)
(5115,)
0.6307210305390171
1.1489833279288122
Traceback (most recent call last):
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 263, in run_path
    pkg_name=pkg_name, script_name=fname)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "bingo_evolution.py", line 237, in <module>
    main(model)
  File "bingo_evolution.py", line 219, in main
    execute_generational_steps(model)
  File "bingo_evolution.py", line 206, in execute_generational_steps
    checkpoint_base_name='checkpoint_', num_checkpoints=5,stagnation_generations=STAGNATION)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/util/argument_validation.py", line 84, in do_validation
    return func(*pargs, **kwargs)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/evolutionary_optimizer.py", line 143, in evolve_until_convergence
    self._update_checkpoints(checkpoint_base_name, num_checkpoints)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/evolutionary_optimizer.py", line 200, in _update_checkpoints
    self._remove_stale_checkpoint()
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/parallel_archipelago.py", line 276, in _remove_stale_checkpoint
    os.remove(self._previous_checkpoints.pop(0))
FileNotFoundError: [Errno 2] No such file or directory: 'checkpoint__0.pkl'
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 0
[unset]: write_line error; fd=-1 buf=:cmd=abort exitcode=1
:
system msg for write_line failure : Bad file descriptor
srun: error: notch272: task 8: Exited with exit code 1
/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evaluation/fitness_function.py:19: RuntimeWarning: overflow encountered in square
  return np.sqrt(np.mean(np.square(vector)))
Traceback (most recent call last):
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 263, in run_path
    pkg_name=pkg_name, script_name=fname)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "bingo_evolution.py", line 237, in <module>
    main(model)
  File "bingo_evolution.py", line 219, in main
    execute_generational_steps(model)
  File "bingo_evolution.py", line 206, in execute_generational_steps
    checkpoint_base_name='checkpoint_', num_checkpoints=5,stagnation_generations=STAGNATION)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/util/argument_validation.py", line 84, in do_validation
    return func(*pargs, **kwargs)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/evolutionary_optimizer.py", line 143, in evolve_until_convergence
    self._update_checkpoints(checkpoint_base_name, num_checkpoints)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/evolutionary_optimizer.py", line 200, in _update_checkpoints
    self._remove_stale_checkpoint()
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/parallel_archipelago.py", line 276, in _remove_stale_checkpoint
    os.remove(self._previous_checkpoints.pop(0))
FileNotFoundError: [Errno 2] No such file or directory: 'checkpoint__0.pkl'
starting
done
[0.98058293 0.94767262 0.91831705 ... 0.42409892 0.44771494 0.47939977]
[[0.90909091 0.2        0.02177   ]
 [0.90909091 0.2        0.20964   ]
 [0.90909091 0.2        0.43674   ]
 ...
 [0.5        0.8        2.69279   ]
 [0.5        0.8        2.91488   ]
 [0.5        0.8        3.13022   ]]
(5115, 3)
(5115,)
0.6307210305390171
1.1489833279288122
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 0
[unset]: write_line error; fd=-1 buf=:cmd=abort exitcode=1
:
system msg for write_line failure : Bad file descriptor
srun: error: notch272: task 4: Exited with exit code 1
starting
done
[0.98058293 0.94767262 0.91831705 ... 0.42409892 0.44771494 0.47939977]
[[0.90909091 0.2        0.02177   ]
 [0.90909091 0.2        0.20964   ]
 [0.90909091 0.2        0.43674   ]
 ...
 [0.5        0.8        2.69279   ]
 [0.5        0.8        2.91488   ]
 [0.5        0.8        3.13022   ]]
(5115, 3)
(5115,)
0.6307210305390171
1.1489833279288122
/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evaluation/fitness_function.py:19: RuntimeWarning: overflow encountered in square
  return np.sqrt(np.mean(np.square(vector)))
Traceback (most recent call last):
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/uufs/chpc.utah.edu/common/home/u6019587/.venv/bingovenv_v2/lib/python3.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 263, in run_path
    pkg_name=pkg_name, script_name=fname)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "/uufs/chpc.utah.edu/sys/installdir/python/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "bingo_evolution.py", line 237, in <module>
    main(model)
  File "bingo_evolution.py", line 219, in main
    execute_generational_steps(model)
  File "bingo_evolution.py", line 206, in execute_generational_steps
    checkpoint_base_name='checkpoint_', num_checkpoints=5,stagnation_generations=STAGNATION)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/util/argument_validation.py", line 84, in do_validation
    return func(*pargs, **kwargs)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/evolutionary_optimizer.py", line 143, in evolve_until_convergence
    self._update_checkpoints(checkpoint_base_name, num_checkpoints)
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/evolutionary_optimizer.py", line 200, in _update_checkpoints
    self._remove_stale_checkpoint()
  File "/uufs/chpc.utah.edu/common/home/u1164746/bingo/bingo/evolutionary_optimizers/parallel_archipelago.py", line 276, in _remove_stale_checkpoint
    os.remove(self._previous_checkpoints.pop(0))
FileNotFoundError: [Errno 2] No such file or directory: 'checkpoint__0.pkl'
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 0
[unset]: write_line error; fd=-1 buf=:cmd=abort exitcode=1
:
system msg for write_line failure : Bad file descriptor
srun: error: notch272: task 2: Exited with exit code 1
slurmstepd: error: *** JOB 3863715 ON notch272 CANCELLED AT 2022-02-04T13:35:48 ***
slurmstepd: error: *** STEP 3863715.0 ON notch272 CANCELLED AT 2022-02-04T13:35:48 ***
